mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
+
+connection slave;
+set global userstat=ON;
```

Comment:
Please save the existing userstat value and restore it in the end (so that MTR --mysqld=--userstat and similar work)